### PR TITLE
feat: Generate app build info after yarn install

### PR DESCRIPTION
**Pull request by ivan-dalmet**
- [x] Generate app build info after `yarn install`
- [x] Remove specific generation in GitHub Action
- [x] Remove specific generation in GitLab CI
- [x] Change the display version from date to short commit hash 
- [x] Rename `app-build` to `build-info`
- [x] Update Dockerfile with git for build

✅ `node vercel.js` removed in Vercel

⚠️ ~~For vercel we will need to remove the `node vercel.js`~~
~~(that's why the preview deploy of this branch will not work)~~

## Steps for this PR
1. ~~Approve this PR~~
2. ~~Remove the `node vercel.js` in vercel~~
3. ~~Trigger preview deploy & test~~
4. Merge PR

### CODE REVIEW

**Comment by yoannfleurydev** [comment]
Related to https://github.com/BearStudio/start-ui-web/pull/211 ?

**Comment by ivan-dalmet** [comment]
> Related to #211 ?

Oh yes :/

**Review by yoannfleurydev — APPROVED** [review]

**Comment by yoannfleurydev** [comment]
Approved, I let you do the other steps @ivan-dalmet as you described in the description.

**Comment by Rileran** [comment]
I'm closing https://github.com/BearStudio/start-ui-web/pull/211 then !

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Install dependencies only when needed
-FROM node:16-alpine AS deps
+FROM timbru31/node-alpine-git:16 AS deps
```
Comment (yoannfleurydev) [inline Dockerfile:2]: Why a non officiel image? What are the benefits of this new one?

Comment (ivan-dalmet) [reply Dockerfile:2]: To be honest, I needed node alpine with git and this is the first result from Google and it works 😅
I will look if I can do it with an official node image.

Comment (ivan-dalmet) [reply Dockerfile:2]: Thanks for pointing it out, I think the fix is pretty easy. I test it and push if it's working.

**Review by yoannfleurydev — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (ivan-dalmet) [reply Dockerfile]: Fixed

Comment (yoannfleurydev) [reply Dockerfile]: Thanks a lot for your justification and time. 👍🏻 

bitmoji

**Pull request merged**